data/story northern union.txt

```diff
@@ -392,72 +392,71 @@ mission "Meeting an Alpha 1"
 			choice
 				`	(Run for your life!)`
 					decline
-				`	"Are you all so tall?"`
+				`	"Are all of you so tall?"`
 					goto tall
 				`	"Yes, lets move on. I heard you were offering work?"`
 					goto work
 			label tall
-			`	He makes that chuckling sound again, "No, in fact some of my kind are even shorter than you. I was created specifically for interrogation and command, unlike my smaller brethren, intended mostly for combat. Now, let's talk about that job."`
+			`	He makes that chuckling sound again, and says: "No. In fact some of my kind are even shorter than you. I was created specifically for interrogation and command, unlike my smaller brethren, intended mostly for combat. Now, let's talk about that job."`
 				goto work
 			label die
-			`	Your death is indeed slow and painful, and everyone around is indifferent to your suffering. Perhaps you shouldn't have mocked the creature who just effortlessly killed a pirate warlord next to you...`
+			`	Your death is indeed slow and painful, and everyone around you is indifferent to your suffering. Perhaps you shouldn't have mocked the creature who just effortlessly killed a pirate warlord next to you...`
 				die
 			label work 
-			`	"The job is an incredibly simple and straight-forward one. Surely you have seen the news reports of the Star Blade, yes? The new Veles class battle cruiser from Betelgeuse? I want you to take some scans of it. I think it's currently patrolling the shipyards where it was constructed. Are you interested?"`
+			`	"The job is an incredibly simple and straight-forward one. Surely you have seen the news reports of the Star Blade, yes? The new Veles class battlecruiser from Betelgeuse? I want you to take some scans of it. I think it's currently patrolling the shipyards where it was constructed. Are you interested?"`
 			choice
-				`	"Simple enough."`
+				`	"Sounds simple enough."`
 					accept
 				`	"What's the worst that could happen?`
 					accept
 				`	"Sorry, but I can't make money when I'm dead."`
 					decline
-				`	"Scan a battle cruiser? Do you think I'm suicidal?"`
+				`	"Scan a battlecruiser? Do you think I'm suicidal?"`
 					decline
 	
 	on accept
 		log "Volunteered to scan the Star Blade, one of the largest vessels in human space."
-		dialog "Upon returning to your ship, you check your communicator, only to find that you have two messages. Upon opening the second one, you find out this belonged to Peg and is full of intel that could be useful to the Republic. Maybe they'll pay you for it?"
+		dialog "Upon returning to your ship, you check your communicator and find that you have received two messages. Upon opening the second one, you find out it belonged to Peg and is full of intel that could be useful to the Republic. Maybe they'll pay you for it?"

```

Comment:
"messages" was meant to be "communicators" 

Two phones essentially. I messed that up.